wa/starter/src/main/java/org/apache/syncope/wa/starter/mapping/DefaultAccessMapper.java

```diff
@@ -43,10 +45,12 @@ public RegisteredServiceAccessStrategy build(final AccessPolicyTO policy) {
         accessStrategy.setUnauthorizedRedirectUrl(conf.getUnauthorizedRedirectUrl());
 
         conf.getRequiredAttrs().forEach(
-                attr -> accessStrategy.getRequiredAttributes().put(attr.getSchema(), new HashSet<>(attr.getValues())));
+                (k, v) -> accessStrategy.getRequiredAttributes().put(k,
+                        new HashSet<>(Set.of(StringUtils.split(v, ",")))));
```

Comment:
Questions:
1. are `new Hashet<>(Set.of(...)) both required? wouldn't be enough to have `Set.of(...)`?
2. can we `trim()` the values after splitting by `,`? I'd like to avoid issues if one inputs `value1, value2` as oppositve as `value1,value2`

Reply:
sure